IndicoV2.StrawberryShake/Reporting/UserReporting.graphql

```diff
@@ -0,0 +1,67 @@
+﻿query UserSummary($date: Date){
+  userSummary(date: $date){
+      users {
+          enabled
+          disabled
+      }
+      appRoles {
+          role
+          count
+      }
+   }
+}
+
+query UserSnapshot($date: Date, $filters: UserReportFilter){
+    userSnapshot(date: $date, filters: $filters){
+        results {
+            id
+            name
+            createdAt
+            enabled
+            roles
+            datasets {
+                datasetId
+                role
+            }
+        }
+        pageInfo{
+            aggregateCount
+        }
+  }
+}
+
+  query UserChangelog($sdate: Date, $edate: Date, $filters: UserReportFilter){
+    userChangelog(startDate: $sdate, endDate: $edate, filters: $filters){
+        results {
+            id
+            date
+            userEmail
+            updaterEmail
+            previousDatasets {
+                datasetId
+                role
+            }
+            changesMade
+        }
+        pageInfo{
+            aggregateCount
```

Comment:
This is fine for now but we should add pagination soon too since this won't get the full report.

Reply:
https://indicodata.atlassian.net/browse/INT-163 <-- there's separate task for pagination